src/app/_client-services/next_api_client_service.ts

```
@@ -832,26 +847,16 @@ export class NextApiClientService {
-		proposalType: EProposalType;
-		content: string;
-		title: string;
-		allowedCommentor: EAllowedCommentor;
-		tags?: ITag[];
-		topic?: EOffChainPostTopic;
-	}) {
+	static async createOffChainPost({ proposalType, allowedCommentor, content, title, tags, topic, poll }: ICreateOffChainPostPayload) {
 		if (!ValidatorService.isValidOffChainProposalType(proposalType)) {
 			throw new ClientError(ERROR_CODES.CLIENT_ERROR, ERROR_MESSAGES[ERROR_CODES.CLIENT_ERROR]);
 		}
 		const { url, method } = await this.getRouteConfig({ route: EApiRoute.CREATE_OFFCHAIN_POST, routeSegments: [proposalType] });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)